src/Symfony/Bundle/Resources/config/doctrine_mongodb_odm.xml

```diff
@@ -156,6 +156,11 @@
             <argument type="service" id="doctrine_mongodb"/>
             <argument type="service" id="api_platform.doctrine_mongodb.odm.metadata.property.identifier_metadata_factory.inner"/>
         </service>
+
+        <service id="api_platform.doctrine_mongodb.odm.graphql_metadata.resource.metadata_collection_factory" class="ApiPlatform\Doctrine\Odm\Metadata\Resource\DoctrineMongoDbOdmResourceCollectionMetadataFactory" decorates="api_platform.graphql_metadata.resource.metadata_collection_factory" decoration-priority="40">
```

Comment:
**issue:** The GraphQL decorated service is not always defined (GraphQL can be disabled).